ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/task/CreateFlowTask.groovy

```diff
@@ -40,9 +40,20 @@ class CreateFlowTask extends HubTask {
                 '  "description" : "Flow description",\n' +
                 '  "steps" : { }\n' +
                 '}\n'
-            def file = ((FlowManagerImpl)flowManager).getFileForLocalFlow(flowName)
+            file = ((FlowManagerImpl)flowManager).getFileForLocalFlow(flowName)
             FileCopyUtils.copy(json.getBytes(), file)
             println "Created new flow at: " + file.getAbsolutePath()
         }
+        ObjectMapper mapper = new ObjectMapper()
+        try{
+            ArtifactService service = ArtifactService.on(hubConfig.newStagingClient(null))
+            service.setArtifact("flow", flowName, mapper.readTree(file))
+        }
+        catch (Exception e){
+            println "Unable to write flow to database;cause: " + e.getMessage()
+            throw e
```

Comment:
I am thinking we should just print the error and not throw it. The error will likely only occur for one reason - the user cannot connect to ML. And the message will make that obvious. The task can still be considered to have succeeded in that a new flow was created.